x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/document_creation_button.test.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { setMockActions } from '../../../__mocks__/kea.mock';
```

Comment:
misc nit, not a blocker
```suggestion
 */

import { setMockActions } from '../../../__mocks__/kea.mock';
```